packages/ui/Remeasurable/Remeasurable.js

```diff
@@ -1,130 +1,13 @@
 /**
- * Exports the {@link ui/Remeasurable.Remeasurable} and {@link ui/Remeasurable.RemeasurableDecorator}
- * higher-order component (HOC). Adds the ability to broadcast remeasure changes
- * based on a callback. The default export is {@link ui/Remeasurable.Remeasurable}.
+ * Exports the Resize context. This is used internally for things to notify children that they need
+ * to resize because of a parent update.
  *
  * @module ui/Remeasurable
```

Comment:
Maybe move this entire thing into `ui/internal/Resize` to keep our "private" code cleanly separated?